engine/src/engine.ts

```diff
@@ -121,7 +121,9 @@ const replaceRegex = new RegExp(
   "g"
 );
 
-const powerRegex = new RegExp(" \\((\\d+/\\d+/\\d+/\\d+) ⇒ \\d+/\\d+/\\d+/\\d+\\)");
+const powerRegex = new RegExp(
+  " \\((\\d+(,B)?/\\d+(,B)?/\\d+(,B)?/\\d+(,B)?) ⇒ \\d+(,B)?/\\d+(,B)?/\\d+(,B)?/\\d+(,B)?\\)"
```

Comment:
would have used `replaceAll` to make the regex shorter - but that's only available in `ESNext`